src/test/java/uk/gov/companieshouse/itemhandler/kafka/OrdersKafkaConsumerTest.java

```diff
@@ -25,96 +31,161 @@
 public class OrdersKafkaConsumerTest {
     private static final String ORDER_RECEIVED_URI = "/order/ORDER-12345";
     private static final String ORDER_RECEIVED_TOPIC = "order-received";
+    private static final String ORDER_RECEIVED_KEY = "order-received";
     private static final String ORDER_RECEIVED_TOPIC_RETRY = "order-received-retry";
     private static final String ORDER_RECEIVED_TOPIC_ERROR = "order-received-error";
+    private static final String PROCESSING_ERROR_MESSAGE = "Order processing failed.";
 
     @Spy
     @InjectMocks
     private OrdersKafkaConsumer ordersKafkaConsumer;
     @Mock
-    private CHKafkaResilientConsumerGroup chKafkaConsumerGroupMain;
+    private OrdersKafkaProducer ordersKafkaProducer;
     @Mock
-    private CHKafkaResilientConsumerGroup chKafkaConsumerGroupRetry;
+    private CHKafkaProducer chKafkaProducer;
     @Mock
     private SerializerFactory serializerFactory;
     @Mock
+    private KafkaListenerEndpointRegistry registry;
+    @Mock
+    private MessageListenerContainer container;
+    @Mock
     private AvroSerializer serializer;
     private static final String EXPECTED_MESSAGE_VALUE = "$/order/ORDER-12345";
```

Comment:
Can we not get the message content and expect that (`ORDER_RECEIVED_URI`)?

Reply:
This has been updated.